src/jbi/whiteboard_actions/README.md

```diff
@@ -1,20 +1,20 @@
 ## Looking to add a new action?
-"actions" are python functions that, once added into `src/jbi/whiteboard_actions/` dir,
+"actions" are python modules that, if available in the PYTHONPATH,
 can be run by updating the `action` attribute of the yaml config file.
 
 ### Create a new action...
 Let's create a `new_action`!
-1. First, add a new python file called "my_team_action" in the `src/jbi/whiteboard_actions/` dir.
+1. First, add a new python file called `my_team_action.py` in the `src/jbi/whiteboard_actions/` directory.
 1. Add the python function `init` to a new "my_team_action" module, let's use the following:
     ```python
     def init(whiteboard_tag, jira_project_key, optional_param=42):
         return lambda payload, context: print("Hi world!")
     ```
```

Comment:
in case we keep it, the `context` is `parameters` here now

Reply:
I like the idea of removing params